File: structs/filajava/Banco.java
public class Banco {
    public static void main(String args[]){
        Fila f;
        int e;
        f = new Fila();

        f.inserir(176);
        f.inserir(914);
        f.inserir(12);
        f.inserir(1002);
        f.inserir(100);

        System.out.println("Bem vindo ao banco xDD!");
        System.out.println("Atendimento");
        while(!f.isEmpty()){
            e = f.retirar();
            System.out.println("Senha a ser atendiade no guiche: " +e);

        }
    }
}

File: structs/filac/banco.c
#include "fila.h"
#include <stdio.h>

int main (int argc, char *argv[]){
    int elemento;
    printf("Bem vindo ao banco xDd!\n");
    struct Fila f;
    create(&f);

    inserir(&f,1);
    inserir(&f,4);
    inserir(&f,3);
    inserir(&f,2);

    
    while (!isEmpty(f))
    {
        elemento = retirar(&f);
        printf("%d ", elemento);
    }

    return 0;
}